SettingsList.py

```diff
@@ -891,6 +949,18 @@ def __init__(self, name, gui_text, min, max, default, step=1,
                     in the Water Temple are not going to be relevant unless this
                     trick is first enabled.
                     '''},
+    'Water Temple Central Pillar GS with Neither Longshot nor Farore\'s Wind': {
```

Comment:
Why not "... with Iron Boots"?

Reply:
That could be a better name. This name is pretty wordy. I don't know how I feel about "with Iron Boots" because a lot of the time the entire water temple is just requiring irons boots, so to say "with Iron Boots" about anything in Water Temple seems self-evident until you realize it's referring to a specific trick that you need the iron boots to do. Does that make any sense? But I've called tricks in the water temple "with Iron Boots" before, so I'm definitely not opposed to doing it.

Part of why I chose to do it like this is I was setting up for potentially turning the FW route into a trick (which I'm definitely leaning in favour of doing), and I wanted the names of the tricks to contrast. So I was thinking "without Longshot using Farore's Wind" (also very wordy) for the new FW trick. So regardless you do have to keep both the Iron Boots and Farore's Wind tricks in mind when deciding names.

Reply:
I think trick names will get pretty wordy if you list both the items you avoid and the items you need. I would just call it "with Farore's Wind" and "with Iron Boots" or whatever. Looking for examples, there's "Dodongo's Cavern Staircase With Bow" doesn't say "without Strength or Bombs using Bow". We don't have to be exact in the name, just make it easy to distinguish what trick a name refers to, and maybe be concise at the same time. So I like it more when we can specify "with Item" or "without Item" to uniquely refer to a trick.

Reply:
At one point I was thinking of having just one trick, "without Longshot" encompass all of it. So in my mind they were both ways of skipping Longshot, so that's why I ended up with "without Longshot" in the name.